src/main/scala/com.socrata.tileserver/config/TileServerConfig.scala

```diff
@@ -19,6 +19,9 @@ object TileServerConfig {
   lazy val renderHost = config.getString("tileserver.render-host")
   lazy val renderPort = config.getInt("tileserver.render-port")
 
+  /** Query configuration. */
+  lazy val queryTimeout = config.getString("query-timeout")
```

Comment:
I know you're going to use this as part of a query string, but consider `.getDuration` instead of `.getString` since that will do some validation.